src/DI/LatteConsoleExtension.php

```diff
@@ -6,45 +6,52 @@
 use Contributte\Console\Extra\Command\Latte\LatteWarmupCommand;
 use Nette\DI\CompilerExtension;
 use Nette\DI\Helpers;
+use Nette\Schema\Expect;
+use Nette\Schema\Schema;
+use stdClass;
 
+/**
+ * @property-read stdClass $config
+ */
 final class LatteConsoleExtension extends CompilerExtension
 {
 
-	/** @var mixed[] */
-	private $defaults = [
-		'warmup' => [
-			'%appDir%',
-		],
-		'warmupExclude' => [],
-		'purge' => [
-			'%tempDir%/cache/latte',
-		],
-	];
+	public static function createSchema(): Schema
```

Comment:
Why static method here?

Reply:
We need schema also in [ConsoleBridgesExtension](https://github.com/contributte/console-extra/blob/next/src/DI/ConsoleBridgesExtension.php#L19)
I can split registration of extensions into two phases (__construct() and loadConfiguration()) to make it non-statical

Reply:
Better to keep it statical. In __construct() we don't know if extension should be instantiated or not.